packages/css/pagination.css

```diff
@@ -8,19 +8,19 @@
     width: fit-content;
   }
 
-  .hidden {
+  .fds-pagination--hidden {
     visibility: hidden;
   }
 
-  .pagination:where(.small) {
+  .fds-pagination--small {
     --fdsc-pagination-chevron-margin: var(--fds-spacing-2);
```

Comment:
Suggest you also fix the css variables in for this component. 

Rename them to `--fds-pagination-*` and declared them in `fds-pagination` width default values.